app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -23,6 +24,10 @@ const menu = [
     name: "Dashboard",
     route: "/",
   },
+  {
+    name: "Map",
+    route: mapRoute,
+  },
```

Comment:
For me the obviously place would be between requests and profile, but no need to change it now.